contrib/seeds/makeseeds.py

```diff
@@ -90,6 +91,10 @@ def filtermultiport(ips):
         hist[ip['sortkey']].append(ip)
     return [value[0] for (key,value) in list(hist.items()) if len(value)==1]
 
+def resolveasn(resolver, ip):
+    asn = int([x.to_text() for x in resolver.query('.'.join(reversed(ip.split('.'))) + '.origin.asn.cymru.com', 'TXT').response.answer][0].split('\"')[1].split(' ')[0])
```

Comment:
Haha, this is a function masquerading as a lambda if you ask me. 😉

Reply:
I this too often in python to be honest... :D
But I didn't want to refactor the "lamba" as well and just copied/moved it